reflect.go

```diff
@@ -1020,6 +1017,19 @@ func ignoredByJSONSchemaTags(tags []string) bool {
 	return tags[0] == "-"
 }
 
+// toJSONNumber converts string to *json.Number
+// it'll return nil if the number is invalid
+func toJSONNumber(s string) *json.Number {
```

Comment:
Is this necessary if pointers are removed? What is the intention here? It would make sense to have a test case for whatever is trying to be caught.

Reply:
Moved from pointers in 372fbaa